python/tests/operations/drop_blocks.py

```diff
@@ -0,0 +1,113 @@
+import os
+import re
+
+import numpy as np
+import pytest
+
+import equistore
+import equistore.io
+from equistore import Labels, TensorMap
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+
+
+@pytest.fixture
+def test_tensor_map() -> TensorMap:
+    return equistore.io.load(
+        os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
+        # the npz is using DEFLATE compression, equistore only supports STORED
+        use_numpy=True,
+    )
+
+
+class TestDropBlock:
+    def test_drop_all(self, test_tensor_map):
```

Comment:
Some short docstrings on each test function describing what the test does would be useful